qcfractal/storage_sockets/models/sql_models.py

```diff
@@ -196,6 +219,13 @@ class KeywordsORM(Base):
     __table_args__ = (Index("ix_keywords_hash_index", "hash_index", unique=True),)
     # meta = {'indexes': [{'fields': ('hash_index', ), 'unique': True}]}
 
+    def to_pydantic_model(self):
```

Comment:
Should ORM classes have this function, even if trivial (like this one for keywords?)

Reply:
Could help with consistency, since the ORM and model diverge slightly it would prevent replication of the alterations throughout.